guide/web-scene.md

```diff
@@ -0,0 +1,71 @@
+## X3D and Web Scene
+
+### Description
+
+Webots can export a world to an interactive 3D `HTML` page.
+This feature is especially useful for publishing Webots-created worlds on the Web.
+
+When doing so, an [X3D](http://www.web3d.org/x3d/what-x3d) file and a `HTML5` page are generated.
```

Comment:
typo: an X3D -> a X3D

Instead of when doing so, I would maybe use 'During the export'